src/runner/main.cpp

```diff
@@ -394,6 +439,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
         // Singletons initialization order needs to be preserved, first events and
         // then modules to guarantee the reverse destruction order.
         modules();
+        cleanup_updates();

```

Comment:
perhaps we should move it into a separate thread, e.g. move before https://github.com/microsoft/PowerToys/pull/19875/files#diff-c07e4e5e9ce3c371d4c47f496b5f66734978a3c4f355c7e446c1ef19e086a4d6R129?